doc/rst/tools/mason/mason.rst

```diff
@@ -608,6 +608,34 @@ command as follows::
 
   mason add MatrixMarket@0.1.0
 
+  
+Specifying Dependencies from `git` Repositories
+===============================================
+To depend on a library located in a git repository, a git key pointing to the repository URL is all that is required:
+
+.. code-block:: text
+
+
+    [dependencies]
+    HelloWorld = { git = "https://github.com/bmcdonald3/HelloWorld" }
+
+
+The repository that is specified must be a mason package itself, but it does not need to be contained in a mason registry.
+
+Without specifying the branch to be used, a default branch of `master` will be assumed and if there is no master branch, the dependency will not work. To specify a different branch, the `branch` key can be used:
```

Comment:
I thought git moved away from `master` in favor of `main` for all new defaults?

Reply:
`master` is still the default name when creating a new git repo.